MdeModulePkg/Library/DxeCorePerformanceLib/DxeCorePerformanceLib.c

```diff
@@ -292,7 +296,10 @@ InternalGetSmmPerfData (
         SmmCommData->Function       = SMM_FPDT_FUNCTION_GET_BOOT_RECORD_SIZE;
         SmmCommData->BootRecordData = NULL;
         Status                      = Communication->Communicate (Communication, SmmBootRecordCommBuffer, &CommSize);
-
+        // MU_CHANGE [BEGIN] - Standalone MM Perf Support
+        ASSERT_EFI_ERROR (Status);

```

Comment:
Can these asserts be changed to DEBUG_ERROR messages?

Reply:
These were in the original code, and I kept the same reporting of status failures here. What is the reason you'd like it changed in this case?

Reply:
You're right. The first MMI should not expect that. I left the asserts on the following MMI (`SMM_FPDT_FUNCTION_GET_BOOT_RECORD_DATA_BY_OFFSET`) since if the first succeeded, it should have given the size necessary for the second not to fail.